vita3k/gui/src/game_selector.cpp

```diff
@@ -43,6 +45,12 @@ void draw_game_selector(GuiState &gui, HostState &host) {
             ImVec2(0, 0), display_size);
     }
 
+    if (gui.delete_game_icon) {
+        if (gui.game_selector.icons[gui.title_id])
```

Comment:
```suggestion
        if (gui.game_selector.icons.find(gui.title_id) != gui.game_selector_icons.end())
```

Using the `[]` operator with maps will create an object if it doesn't exist (which is not really good).

Reply:
I not very like that, get code more long for nothing 🤔

Reply:
I hate C++ for this. I would rather a `.contains()` method rather than an `.empty()` method.

But the `[ ]` has side-effects. It's worth the change :)

Reply:
i see, but what second effect ? ^^ 
for me here want just said check if game is found inside gui.game_selector.icons[gui.title_id]

Reply:
Talked about this in dms but will put it here for reference.
`[ ]` will create an element for each input. It kinda defeats the purpose of checking if it exists (because it will exist after you use it). Thankfully, it still works because the new element is `nullptr`- but it's still not good to have copies of random stuff anyway.

Reply:
i have done like you want with test game with and without icon, it found or not found, i have pushed